python/ray/tests/test_metrics_agent.py

```diff
@@ -643,7 +645,7 @@ def verify():
             assert {"raylet", "gcs_server"} == components
         return True
 
-        wait_for_condition(verify, timeout=30)
```

Comment:
hmm, how did it work before

Reply:
I think it just wasn't testing anything 😅 because this was inside the scope of the `verify` fn definition